sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseExportDto.java

```diff
@@ -681,6 +683,16 @@ public boolean isQuarantineExtended() {
 		return quarantineExtended;
 	}
 
+	@Order(137)
```

Comment:
also the @Order should be changed to `42` and increase the ones following

@Order annotation sets the position of the column in the exported CSV. 

`quarantineExtended`, `quarantineReduced` fields should go one after another.